Makefile

```diff
@@ -101,19 +101,16 @@ build-catalog-json: build-tnf-tool
 	./tnf generate catalog json > catalog.json
 
 # generate the test catalog in Markdown
-build-catalog-md: build-tnf-tool
+build-catalog-md: build-tnf-tool classification-js
```

Comment:
Some comments here:
- Where is "make build-catalog-md" done now?
- Is "classification-js" actually needed for building the CATALOG.md? Otherwise I don't think it should be a dependency here. In fact, the "classification-js:" rule (line 162) should actually have the "build-tnf-tool" rule as a dependency (like build-catalog-md).
- Maybe we should include both "make build-catalog-md" and "make classification-js" under the "build:" rule or creating a new one like "artifacts:", or "docs:".